openshift/pipelines/Jenkinsfile.cicd

```diff
@@ -47,6 +48,12 @@ pipeline {
     stage("Initialize") {
       agent any
       steps {
+        // Cancel any running builds in progress
+        timeout(10) {
+          echo "Cancelling previous ${APP_NAME} builds in progress..."
+          abortAllPreviousBuildInProgress(currentBuild)
```

Comment:
Did you test this or copied from other pipelines in bcgov? I ask because I'm not sure we import that shared lib (yet)

Reply:
After testing I ran into Jenkins permission issues with this feature.  I've since removed it.